### PR TITLE
[APPS-2916] Add release job

### PR DESCRIPTION
**Pull request by SathishK-T**
https://hyland.atlassian.net/browse/APPS-2916

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
       - develop
       - release/**
       - dev-**
+      - fix/**
```
Comment (suneet-gupta) [inline .github/workflows/ci.yml:9]: I think we can remove fix/** branch

File: pom.xml [hunk 2]
```diff
@@ -52,9 +52,10 @@
     <module>integration-tests</module>
   </modules>
   <scm>
-    <url>http://github.com/${project.scm.organisation}/${project.scm.repository}</url>
-    <connection>scm:git:git://github.com/${project.scm.organisation}/${project.scm.repository}.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/${project.scm.organisation}/${project.scm.repository}.git</developerConnection>
+    <connection>scm:git:https://github.com/Alfresco/alfresco-java-sdk.git</connection>
+    <developerConnection>scm:git:https://github.com/Alfresco/alfresco-java-sdk.git
+    </developerConnection>
```
Comment (suneet-gupta) [inline pom.xml]: We can make closing tag at the same line

Comment (SathishK-T) [reply pom.xml]: done the changes.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (SathishK-T) [reply .github/workflows/ci.yml:9]: done.

**Review by omkaryadv177 — APPROVED** [review]
LGTM

**Review by suneet-gupta — APPROVED** [review]
LGTM

**Pull request merged**